app/src/controllers/plugins/uiExtensions/sagas.js

```diff
@@ -63,7 +65,9 @@ export function* fetchUiExtensions() {
   }
 
   const plugins = yield select(pluginsSelector);
-  const uiExtensionPlugins = plugins.filter(
+  const publicPlugins = yield select(publicPluginsSelector);
+  const allPlugins = plugins.concat(publicPlugins);
+  const uiExtensionPlugins = allPlugins.filter(
```

Comment:
This is also unnecessary, because there are a legacy extensions and we should avoid using them in new plugins.